fog/ledger/server/tests/router_integration.rs

```diff
@@ -115,22 +134,24 @@ fn seed_block_provider(block_provider: &mut LocalBlockProvider<LedgerDB>) {
     add_block_to_ledger(block_provider, &recipients, &[], &mut rng);
 }
 
-fn populate_block_provider(
+fn populate_block_provider<'a>(
     block_provider: &mut LocalBlockProvider<LedgerDB>,
-    blocks_config: &BlockConfig,
+    blocks_config: impl IntoIterator<Item = &'a HashMap<PublicAddress, Vec<KeyImage>>>,
```

Comment:
Curious about this change - will come back after understanding more of the PR as a whole

Reply:
I rediscovered why!
Previously `BlockConfig` was a vector, this forced one to always create a `vec![blah]` when calling this, by making this take a generic iterator, one can pass a vector, or a `[block1, block2]` directly
basically makes it a bit more ergonomic for the callers